internal/pkg/template/workload.go

```diff
@@ -420,6 +420,7 @@ type DeadLetterQueue struct {
 type NetworkOpts struct {
 	AssignPublicIP string
 	SubnetsType    string
+	SubnetIDs      []string
```

Comment:
nit: can you add a comment here mentionining `SubnetsType` and `SubnetIDs` are mutually exclusive